### ISSUE TITLE
Biospecimen Dashboard- Updates to collection and shipping locations (Biospecimen) (September Release)

### ISSUE THREAD

**Issue by wyattks** [episphere/connect#1008]
**UChicago updates:**

Please remove **River East** and **South Loop** from the Shipping Location dropdown located on the Packaging screen.

Chicago is converting these two locations to Clinical and will be shipping all specimens from a central location.

Also, please remove **Orland Park** from Collection Location dropdown on the Specimen Link screen and the Shipping Location dropdown located on the Packaging screen. This site will no longer be a research collection location.

**Comment by erincschwartz** [comment]
Also, please remove Orland Park from Collection Location dropdown on the Specimen Link screen and the Shipping Location dropdown located on the Packaging screen. This site will no longer be a research collection location.

**Comment by erincschwartz** [comment]
**Henry Ford Updates:** 

Please remove **Henry Ford West Bloomfield Hospital** and **Henry Ford Medical Center-Fairlane** from Henry Ford's list of Shipping Locations located on the Packaging screen.

**Comment by amber-emmes** [comment]
These locations are controlled by the site locations available in Firestore. This will require updates to the actual database. I believe you'll want to talk to @mnataraj92 about this?

**Comment by brotzmanmj** [comment]
Hi @amber-emmes  We don't want to delete them in Firestore; they are valid locations for samples already collected and shipped. We just want to remove them from appearing on the dropdowns on the Biopecimen Dashboard UI so the user can't select them going forward.

**Comment by amber-emmes** [comment]
We will probably want to discuss how to reflect that in the data in the long term, but in the short term I can hard code that in, then.

**Comment by amber-emmes** [comment]
This should now be in dev and ready to test.

**Comment by erincschwartz** [comment]
Kirsti from HFH confirmed that Henry Ford West Bloomfield Hospital and Henry Ford Medical Center-Fairlane are no longer options in Dev on Henry Ford's list of Shipping Locations. **This change is ready for Stage**. 

Waiting on reply from UChicago.

**Comment by erincschwartz** [comment]
UChicago reports that Orland Park is still showing up as a colleciton location in Dev. Please look into this.

Orland Park, River East and South Loop have been removed from Shipping Location in Dev, as expected.

**Comment by amber-emmes** [comment]
Please re-review this in dev now.

**Comment by erincschwartz** [comment]
UChicago just re-tested. Orland Park is now removed from Collection Location list and all 3 locations are still removed from Shipping Location list. **This change is ready for Stage.** 

Waiting on HFH to test and will update here when complete.

**Comment by erincschwartz** [comment]
@amber-emmes Kirsti reported this morning that Henry Ford Medical Center-Fairlane is back in the Shipping Locations list. Please look into this. 

She reports that Henry Ford West Bloomfield Hospital is still removed from the list, as expected.

**Comment by amber-emmes** [comment]
Looks like I was filtering out 'Henry Ford Medical Center-Fairlane' while the name in the database was actually 'Henry Ford Medical Center- Fairlane'. Should be fixed in dev now; please have her re-review.

**Comment by erincschwartz** [comment]
Stage testing completed by UChicago, their Collection and Shipping Location changes are ready for Prod. 

Waiting on HFH to re-test in Dev.

**Comment by erincschwartz** [comment]
HFH just completed Dev re-test, looks good now. Both Henry Ford West Bloomfield Hospital and Henry Ford Medical Center-Fairlane are removed from Shipping Location list. 

Should I have them test this in Stage? Or will today's updates need to be merged?

**Comment by amber-emmes** [comment]
Today's updates will need merged; I'll submit a PR.

**Comment by amber-emmes** [comment]
This is now in stage and ready to test there.

**Comment by erincschwartz** [comment]
@amber-emmes Just heard back from Kirsti, Fairlane location is still showing up in the Shipping Locations list in Stage. Please investigate.

**Comment by amber-emmes** [comment]
It appears that the name in stage is 'Henry Ford Medical Center-Fairlane' while the name in dev is 'Henry Ford Medical Center- Fairlane'. I'm going to just check if the name contains both "Henry Ford Medical Center" and "Fairlane" for now in a new PR.

**Comment by amber-emmes** [comment]
Please re-test in dev and once it's confirmed I'll get it to stage.

**Comment by erincschwartz** [comment]
Just confirming, Kirsti already re-tested in Dev earlier today, as noted above and Fairlane was missing. The issue arose in Stage. Do I need to go back to her to re-test in Dev?

**Comment by amber-emmes** [comment]
Yeah, please re-test in dev as a sanity check.

**Comment by wyattks** [comment]
Kirsti verified Dev is fine now

**Comment by amber-emmes** [comment]
Okay, please re-test in stage.

**Comment by wyattks** [comment]
Ready for Prod
